spec/lib/audit/checksum_spec.rb

```diff
@@ -11,6 +11,17 @@
   let(:endpoint_name) { 'fixture_sr1' }
   let(:limit) { Settings.c2m_sql_limit }
 
+  describe '.logger' do
+    let(:logfile) { Rails.root.join('log', 'cv.log') }
+
+    after { FileUtils.rm_f(logfile) }
+
+    it 'writes to STDOUT and its own log' do
+      expect { described_class.logger.debug("foobar") }.to output(/foobar/).to_stdout_from_any_process
```

Comment:
I guess `to_stdout_from_any_process` is significantly (~30x) slower than `to_stdout`.
I forget, did we try `to_stdout` here?

Reply:
We tried the `to_stdout` matcher, but then this expectation wouldn't pass, even though we can demonstrate the logger writing to STDOUT:

```
[1] pry(main)> Audit::CatalogToMoab.logger.debug("foobar")
D, [2018-07-06T14:50:14.472858 #36733] DEBUG -- : foobar
```

This matcher is slower, but I don't detect a significant lag when the test suite runs, and it seems to pick up on writing to STDOUT.

Reply:
Sounds good.